code/modules/surgery/limb_augmentation.dm

```diff
@@ -42,9 +42,9 @@
 		for(var/part_name in L.part)
 			if(!target.get_organ(part_name))
 				continue
-			affected.robotize(L.model_info, make_tough = 1, convert_all = 0)
+			affected.robotize(L.model_info, make_tough = TRUE, convert_all = FALSE)
```

Comment:
May i ask to change `robotize` signature as well to have `TRUE`/`FALSE` rather than `1`/`0`?

Reply:
Missed that one, done!